### DISCUSSION TITLE
O GitHub na relação o código local do Brasil +55 não consta para habilitação de 2 fatores!

### DISCUSSION THREAD

**Discussion by feccruz** [community/community#48709] [Mobile]
### Select Topic Area

Bug

### Body

Tentei habilitar o meu celular para de 2 fatores e na relação não  consta o código do Brasil +55!
Que é o responsável pela atualização do Sistema GitHub por favor corrigir este erro!

**Comment by feccruz** [comment]
Tentei habilitar o meu celular para de 2 fatores e na relação não  consta o código do Brasil +55!
Que é o responsável pela atualização do Sistema GitHub por favor corrigir este erro!

**Reply by feccruz** [reply]
Quem pode me ajudar com  este problema e erro do sistema GitHub?

Tentei habilitar o meu celular para de 2 fatores e na relação não  consta o código do Brasil +55!
Que é o responsável pela atualização do Sistema GitHub por favor corrigir este erro!

**Comment by elaversannio** [comment]
Estou com o mesmo problema, não consigo habilitar o meu celular para verificação, pois não consta o código do Brasil +55... acho isso um absurdo!
